internal/store/digest.go

```diff
@@ -0,0 +1,54 @@
+package store
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/distribution/reference"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+// We store bundles both by image:tags and by plain digest.
+//
+// Within the bundle store, the file layout is
+// <registry>
+//  \_ <repo>
+//       \_ <tag>
+//           \_ bundle.json
+//              digest    # plain text file containing bundle's digest
```

Comment:
Can't we just recompute the digest using the `bundle.json` itself?

Reply:
indeed